packages/rich-text-editor/src/EditableRichTextContent/EditableRichTextContent.scss

```diff
@@ -0,0 +1,27 @@
+@import "@kaizen/design-tokens/sass/animation";
+@import "@kaizen/design-tokens/sass/border";
+@import "@kaizen/design-tokens/sass/color";
+@import "@kaizen/design-tokens/sass/spacing";
+
+.editableContainer :global(.ProseMirror) {
+  padding: $spacing-sm;
+  position: relative;
+  border-radius: $border-solid-border-radius;
+  border: $border-focus-ring-border-width $border-focus-ring-border-style
+    transparent;
+  transition: background-color $animation-duration-immediate,
+    border-color $animation-duration-immediate;
+}
```

Comment:
I was thinking we should have a border when its inactive just to make it clear its an input? That is unless a design says otherwise, in which case feel free to ignore 😆

Reply:
Nice 👍